lux/utils/utils.py

```diff
@@ -92,8 +92,16 @@ def check_if_id_like(df, attribute):
             and str_length_uniformity
         )
     else:
-        # TODO: Could probably add some type of entropy measure (since the binned id fields are usually very even)
-        return high_cardinality and (attribute_contain_id or almost_all_vals_unique)
+        if len(df) >= 2:
+            series = df[attribute]
+            diff = series.diff()
+            evenly_spaced = all(diff.iloc[1:] == diff.iloc[1])
+        else:
+            evenly_spaced = True
+        if attribute_contain_id:
+            almost_all_vals_unique = df.cardinality[attribute] >= 0.75 * len(df)
+            return high_cardinality and (almost_all_vals_unique or evenly_spaced)
```

Comment:
this line can be deleted

Reply:
Do you want to delete all lines from 100 to 103?

Reply:
@jerry - Done the required changes and pushed as well.